test/quicklink.spec.js

```diff
@@ -34,10 +34,10 @@ mainSuite('should prefetch in-viewport links correctly (UMD)', async () => {
   });
   await page.goto(`${server}/test-basic-usage.html`);
   await sleep(1000);
-  assert.equal(Array.isArray(responseURLs), true);
```

Comment:
Or can use another method :P

```suggestion
  assert.instance(responseURLs, Array);
```

Reply:
I didn't see this documented and being that Arrays are treated weird in JS I thought you didn't cover this, but indeed you cover it with `constructor.name`. :)

Reply:
It is :) just passes thru an instanceof check